step_by_step_install_of_workadventure.md

```diff
@@ -0,0 +1,89 @@
+# step_by_step_install_of_workadventure
+this is a step by step install for workadventure from a public pad (debianbased):
+https://pad.riseup.net/p/step_by_step_prod
+
+this is an attempt to gather points for a step by step installation for a prod. env.
+feel free to add your points!!!!!!!11!1
+
+### 1.0 installation on debian(based):
+maybe with sudo or in rootmode (e.g. sudo -i, su -)
+* `apt update & apt upgrade`
+* `apt install git curl wget ufw` 
+* `apt install docker docker-compose`
+
+### 1.1 the user / directory:
+* e.g. `adduser --disabled-login --shell /bin/bash workadventure`
+* e.g. `cd /home/workadventure/`
+
+### 1.2 download workadventure:
+* `git clone -b master --single-branch https://github.com/thecodingmachine/workadventure/tree/master`
```

Comment:
You probably don't need to clone the project at all (only downloading docker-compose.prod.yml as described in 1.3 is enough).